src/operatorApp/tests/operator.test.js

```diff
@@ -180,4 +180,140 @@ describe('Operator router tests', () => {
 			.set('Authorization', `Bearer ${token}`);
 		expect(response).to.have.status(400);
 	});
+	// Testing for logged in as an operator
+	it('should login an operator user', async () => {
+		const password = await bcrypt.hash('Andela@1234', 5);
+		const newUser = User.create({
+			email: 'operator@gmail.com',
+			password,
+			firstName: 'test',
+			lastName: 'Andela',
+			role: 'operator',
+		});
+		const user = await newUser.save();
+		const info = { email: user.email, password: 'Andela@1234' };
+		const response = await chai
+			.request(app)
+			.post('/api/v1/accounts/login')
+			.send(info);
+		expect(response).to.have.status(200);
+		token = response.body.data.access_token;
+	});
+	it('should not create an operator', async () => {
+		operatorInfo = {
+			firstName: 'testName',
+			lastName: 'testname2',
+			email: 'tiktok@me.com',
+			mobileNumber: '0788352746',
+			company: 'Virunga express',
+			address: 'Kabuga, Kigali',
+			nationalID: '1199880081691164',
+		};
+		const response = await chai
+			.request(app)
+			.post('/api/v1/operators')
+			.send(operatorInfo)
+			.set('Authorization', `Bearer ${token}`);
+		expect(response).to.have.status(403);
+	});
+	it('should not create an operator', async () => {
+		operatorInfo = {
```

Comment:
create an object in that file and call it everywhere you're using it

Reply:
I am working on those fixes right now